### PR TITLE
Add a couple more links

### PR DESCRIPTION
**Pull request by SimenB**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 ### Matchers
 
 - [jest-extended](https://www.github.com/jest-community/jest-extended) Adds additional matchers to core API making it easy to test everything.
+- [expect-more](https://github.com/JamieMason/expect-more/tree/master/packages/expect-more-jest)
```
Comment (SimenB) [inline README.md]: @JamieMason what do you want as text here?

File: README.md [hunk 2]
```diff
@@ -31,12 +32,17 @@
...
+- [jest-runner-mocha](https://github.com/rogeliog/jest-runner-mocha) Mocha runner for Jest.
 
 ### Snapshot
 
 - [snapshot-diff](https://www.github.com/jest-community/snapshot-diff) Takes two values, and return their difference as a string, ready to be snapshotted with toMatchSnapshot(). Especially helpful when testing the difference between different React component states.
 - [jest-snapshots-svg](https://www.github.com/jest-community/jest-snapshots-svg) Take a React component tree, and render it into an SVG.
 
+### Migration
+
+- [jest-codemods](https://github.com/skovhus/jest-codemods)
```
Comment (SimenB) [inline README.md]: @skovhus ideas for text? 🙂

Comment (orta) [reply README.md]: "Makes it easy to migrate to and from Jest from other test runners"

Comment (SimenB) [reply README.md]: I'll take it 🙂

File: README.md (hunk 1 above)
Comment (JamieMason) [reply README.md]: Thanks @SimenB. "A huge library of test matchers for a range of common use-cases" from the readme would be great, thank you.

Comment (SimenB) [reply README.md]: Done!

**Comment by SimenB** [comment]
Whoops, forgot the ToC

File: README.md (hunk 2 above)
Comment (skovhus) [reply README.md]: As we don’t migrate from Jest (yet) I guess something like

“Makes it easy migrate from other test runner and frameworks to Jest.”

And thanks for updating this!

Comment (SimenB) [reply README.md]: Ah, that's a very good point 😅

**Pull request merged**